exaudfclient/base/python/python3/python_ext_dataframe.cc

```diff
@@ -417,15 +476,17 @@ inline void getColumnTypeInfo(PyObject *numpyTypes, std::vector<std::pair<std::s
 
 }
 
-#ifdef NDEBUG
 inline void printPyObject(PyObject* obj, const std::string& error_code){
         PyTypeObject* type = obj->ob_type;
         const char* p = type->tp_name;
         PyObject* objectsRepresentation = PyObject_Repr(obj);
         const char* s =  PyUnicode_AsUTF8(objectsRepresentation);
-        throw std::runtime_error(error_code+": "+std::string(s)+" "+std::string(p));
+        DBG_STREAM_MSG(std::cerr, error_code << ": " << std::string(s) << " " << std::string(p));
+}
+
+inline bool isNoneOrNA(PyObject* pyVal){
+    return pyVal == Py_None  || std::string(Py_TYPE(pyVal)->tp_name) == "NAType";
```

Comment:
check if we can get the value of pd.NA and compare against this, instead of using type()==NAType